Mountain_Duck/Preferences.md

```diff
@@ -80,6 +80,40 @@ You must select NTFS formatted drives with support for *NTFS Alternate Data Stre
 ````
 `````
 
+# Preferences → Profiles
+
+```{note}
+Prelimitary documentation for Mountain Duck version 4.8
+``` 
+
+Select connection profiles to be installed. Either scroll through the list or use the search function to look for a specific profile. The connection profiles will be installed after ticking the corresponding checkboxes. Installed protocols are displayed in the _Protocol_ section of the bookmark window. To disable the connection profile simply untick the checkbox. The profile will be disabled after closing the application.
```

Comment:
>Installed protocols are displayed in the _Protocol_ section of the bookmark window.

Misleadingly formulated, since there is no field in the bookmark configuration window that is named _Protocol_.